h/h_api/resources/schema/json_api.json

```diff
@@ -56,9 +55,11 @@
                 "id": {"type": "string"},
                 "type": {"type": "string"},
                 "attributes": {"type": "object"},
+                "relationships": {"$ref": "#/$defs/relationships"},
```

Comment:
This was nested at the wrong level. It's `{"data": {"relationships": {...}}` not a root item